typescript/forms.d.ts

```diff
@@ -497,7 +500,17 @@ export type PointAddressElement = {
   isDisplayingAddressInformation?: boolean
 } & LookupFormElement &
   FormElementRequired &
-  FormElementReadOnly
+  FormElementReadOnly &
+  (
+    | {
+        defaultValue?: PointAddress
+        version?: 'v2'
+      }
+    | {
+        defaultValue?: PointAddressResponseV3
+        version: 'v3'
+      }
+  )
```

Comment:
Can we please make this two elements. Dealing with submission data differently based on properties other than `type` is a bit of a hassle in some code bases.

Reply:
I'm not seeing how... Should just be a sub switch statement within the `point` case?

Reply:
Happy to change if you can provide a compelling example.

Reply:
ive got nothing objective